packages/jest-cli/src/SnapshotInteractiveMode.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @flow
+ */
+
+import type {AggregatedResult} from 'types/TestResult';
+
+const chalk = require('chalk');
+const ansiEscapes = require('ansi-escapes');
+const {pluralize} = require('./reporters/utils');
+const {rightPad} = require('./lib/terminalUtils');
+const {KEYS} = require('./constants');
+
+module.exports = class SnapshotInteractiveMode {
+  _pipe: stream$Writable | tty$WriteStream;
+  _isActive: boolean;
+  _updateTestRunnerConfig: (a: string, jestRunnerOptions: Object) => *;
+  _testFilePaths: Array<string>;
+  _countPaths: number;
+
+  constructor(pipe: stream$Writable | tty$WriteStream) {
+    this._pipe = pipe;
+    this._isActive = false;
+  }
+
+  isActive() {
+    return this._isActive;
+  }
+
+  _drawUIOverlay() {
+    this._pipe.write(ansiEscapes.scrollDown);
+    this._pipe.write(ansiEscapes.scrollDown);
+
+    this._pipe.write(ansiEscapes.cursorSavePosition);
+    this._pipe.write(ansiEscapes.cursorTo(0, 0));
+
+    const title = rightPad(' -> Interactive Snapshot Update Activated <-');
```

Comment:
Also, this text appears in "random" position. 

<img width="493" alt="screen shot 2017-06-29 at 13 32 10" src="https://user-images.githubusercontent.com/5106466/27685635-631ce760-5ccf-11e7-95cc-e403f3e3dde8.png">

Can we put it on top? Or even resign, as we have information about Interactive Snapshot Progress below.